languages/csharp/exercises/concept/constructors/.meta/design.md

```diff
@@ -0,0 +1,44 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the Concept of Constructors in C#.
+
+## Learning objectives
+
+- Know what constructors are.
```

Comment:
Nit: do we want periods here since these aren't really complete sentences but more phrases?

Reply:
Yeah that makes sense.